Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -383,7 +378,16 @@ view push state =
             -- , if state.props.zoneTimerExpired then zoneTimerExpiredView state push else emptyTextView state
             , if state.props.callSupportPopUp then callSupportPopUpView push state else emptyTextView state
             , if state.props.showDisabilityPopUp &&  (getValueToLocalStore DISABILITY_UPDATED == "true") then disabilityPopUpView push state else emptyTextView state
-            ] <> if state.props.showEducationalCarousel then 
+            , if state.props.repeatRideTimer /= "0" 
+              then linearLayout
+                    [ width MATCH_PARENT
+                    , height MATCH_PARENT
+                    , onClick push $ const StopRepeatRideTimer
+                    , clickable $ not DS.null state.props.repeatRideTimerId 
+                    , visibility if not DS.null state.props.repeatRideTimerId then VISIBLE else GONE
```

Comment:
Better to have  `state.props.repeatRideTimer /= "0"`  and `not DS.null state.props.repeatRideTimerId` at one place Since both determines view's visibility.
Instead of having at two places